Service/NavitiaService.php

```diff
@@ -68,7 +68,7 @@ public function processConfiguration($config)
      */
     public function process($query, $format = null, $timeout = 6000, $pagination = true)
```

Comment:
If you can override the config by passing a timeout to the function, you should do it.
This function is called by the serviceFacade (you should set the timeout to null by default and if null, used the config one).